pyogrio/tests/test_raw_io.py

```diff
@@ -219,3 +220,49 @@ def test_write_unsupported(tmpdir, naturalearth_lowres):
 
     with pytest.raises(DriverError, match="does not support write functionality"):
         write(filename, geometry, field_data, driver="OpenFileGDB", **meta)
+
+
+def assert_equal_result(result1, result2):
+    meta1, geometry1, field_data1 = result1
+    meta2, geometry2, field_data2 = result2
+
+    assert np.array_equal(meta1["fields"], meta2["fields"])
+    # assert np.array_equal(geometry1, geometry2)
+    import pygeos
+    assert pygeos.equals_exact(
+        pygeos.from_wkb(geometry1), pygeos.from_wkb(geometry2), tolerance=0.00001
+    ).all()
```

Comment:
I know we don't strictly require pygeos for the base package, but the problem here is that for a certain format (in this case GeoJSON) the coordinates changed slightly, so asserting based on the WKB fails.

Reply:
I think that is fine to use pygeos in tests.